Library/Formula/libcppa.rb

```diff
@@ -2,8 +2,11 @@ class Libcppa < Formula
   # TODO: since libcppa has been renamed to CAF, this formula should eventually
   # be renamed to 'caf.rb'.
   homepage "http://actor-framework.org/"
-  url "https://github.com/actor-framework/actor-framework/archive/0.13.tar.gz"
-  sha1 "80de2666cad9795f280994f6dd12aae8368e65fe"
+  # For this release (0.13.1) we do not use a github tag because we've added
+  # mountain-lion compatibility fix discovered by brew-bot.
```

Comment:
Because the `0.13.1` here is not the same as the `0.13.1` tag so it doesn't make sense to me to label it as that. Additionally, it may make other formula authors think that it's OK to make formulae on untagged commits. Perhaps instead we could use the `0.13.1` release tag and apply a patch?